src/services/api/actions/__tests__/getDrugDefinition.test.js

```diff
@@ -0,0 +1,16 @@
+import { setAPIEndpoint } from '../../endpoints';
+import { getDrugDefinition } from '../index';
+
+describe('getDrugDefinition action', () => {
+	setAPIEndpoint('/api/drugdictionary/v1/');
+
+	test(`should match getDrugDefinition action for drug "bevacizumab"`, () => {
+		const idOrName = 'bevacizumab';
+		const retAction = {
+			method: 'GET',
+			endpoint: `/Drugs/${encodeURIComponent(idOrName)}`,
```

Comment:
Please do not use functions in tests. If encodeURIComponent were to ever not work as you expect, this test would still pass.